packages/framework/src/Migrations/Version20240910054629.php

```diff
@@ -30,31 +30,31 @@ public function up(Schema $schema): void
             $this->sql(
                 'UPDATE mail_templates SET subject = :body WHERE name = \'complaint_status_1\' AND domain_id = :domainId',
                 [
-                    'body' => t('Complaint status has changed', [], Translator::DEFAULT_TRANSLATION_DOMAIN, $domainLocale),
+                    'body' => t('Status of complaint with number {complaint_number} has changed', [], Translator::DEFAULT_TRANSLATION_DOMAIN, $domainLocale),
```

Comment:
this will be necessary to do manually after merge in our prod/devel environments as this migration was already applied